broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -1607,7 +1607,7 @@ private RemotingCommand resumeCheckHalfMessage(ChannelHandlerContext ctx,
                 response.setRemark("Put message back to RMQ_SYS_TRANS_HALF_TOPIC failed.");
             }
         } catch (Exception e) {
-            log.error("Exception was thrown when putting message back to RMQ_SYS_TRANS_HALF_TOPIC.");
+            log.error("Exception was thrown when putting message back to RMQ_SYS_TRANS_HALF_TOPIC.",e);
             response.setCode(ResponseCode.SYSTEM_ERROR);
             response.setRemark("Exception was thrown when putting message back to RMQ_SYS_TRANS_HALF_TOPIC.");
         } finally {
```

Comment:
It would be better to add exception information to the response

Reply:
good idea

Reply:
I mean add both

Reply:
I have updated